docs/diagrams/OptimisticDACertificateRecv.md

```diff
@@ -0,0 +1,8 @@
+# OptimisticDACertificateRecv
```

Comment:
Are we going to distinguish the optimistic DAC from the normal (or other types of) DAC? We were previously, but not in the current code, so I'm curious about the terminology change here.

Reply:
I figured this was a more descriptive name for the DAC.  (Actually you were the one who suggested it a while ago!).  I think it is what we call it in the HotShot paper too, so I figured it would be good to match that.